var/spack/repos/builtin/packages/gosam-contrib/package.py

```diff
@@ -26,6 +29,11 @@ class GosamContrib(AutotoolsPackage):
     )
     variant("pic", default=False, description="Build position-independent code")
 
+    def patch(self):
+        # remove spack compiler wrapper path
+        mf = FileFilter("gosam.conf.in")
+        mf.filter("^fc.bin=.*", "fc.bin=gfortran")
```

Comment:
Can we use e.g. `self.compiler.fc` instead of hardcoding `gfortran`?

Reply:
Yes, good call. Took a while, but I tested it now.
```suggestion
        mf.filter("^fc.bin=.*", "fc.bin=" + self.compiler.fc)
```